src/Native/Runtime/gcrhenv.cpp

```diff
@@ -1493,6 +1496,19 @@ bool GCToEEInterface::CreateThread(void (*threadStart)(void*), void* arg, bool i
     return true;
 }
 
+void GCToEEInterface::WalkAsyncPinnedForPromotion(Object* object, ScanContext* sc, promote_func* callback)
+{
+}
+
+void GCToEEInterface::WalkAsyncPinned(Object* object, void* context, void (*callback)(Object*, Object*, void*))
+{
+}
```

Comment:
Is this correct (it's what the GC sample does too)?
It seemed to me that in the GC, all code related to this was disabled on `FEATURE_REDHAWK`.

Reply:
Yes, that's correct. CoreRT does not use the async pinned handles. This method should be unrechable in CoreRT - if you would like, you can add assert or comment to make it more obvious.